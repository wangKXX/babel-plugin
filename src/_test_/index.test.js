const babel = require('babel-core');
const plugin = require('../');

var example = `
const a = 1;
console.log(a);
debugger;
if (DEBUG) {
  console.log(2222)
}
console.log(7);
console.info(8);
`;

it('works', () => {
  const {code} = babel.transform(example, {plugins: [plugin]});
  expect(code).toMatchSnapshot();
});

it('works', () => {
  const {code} = babel.transform(example, {plugins: [[plugin, { noDebugger: true }]]});
  expect(code).toMatchSnapshot();
});