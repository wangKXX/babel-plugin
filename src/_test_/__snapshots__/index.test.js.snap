// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"
const a = 1;

debugger;
if (true) {}"
`;

exports[`works 2`] = `
"
const a = 1;

if (true) {}"
`;
